### PR TITLE
tools: don't display command not found

### PR DESCRIPTION
**Pull request by julianoes**
This just caught my eye.

### CODE REVIEW

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Hmm which terminal did you test it with? For me with zsh and sh:

sh:
```
$ type podman > /dev/null
$ type podman 2> /dev/null
podman: not found
```

zsh:

```
% type podman > /dev/null
% type podman 2> /dev/null
podman not found
```

So it seems to me that `>` does what I expect, and `2>` does not :thinking:

**Comment by julianoes** [comment]
odd, I tested it in zsh.

**Comment by julianoes** [comment]
```
tools/run-docker.sh tools/fix_style.sh .
tools/run-docker.sh: line 5: type: podman: not found
Starting with UID: 1000
```

**Comment by julianoes** [comment]
Bash:
```
type podman > /dev/null
bash: type: podman: not found
```

**Comment by julianoes** [comment]
And since this is made for bash I think the PR is correct.

File: tools/run-docker.sh [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 dockerimage=mavsdk/mavsdk-ubuntu-20.04-px4-sitl-v1.11
 
-if type podman > /dev/null
+if type podman 2> /dev/null
```
Comment (JonasVautherin) [inline tools/run-docker.sh:5]:
```suggestion
if type podman > /dev/null 2> /dev/null
```

File: tools/run-docker.sh [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 then
     podman run -it --rm -v $(pwd):/home/user/MAVSDK:z $dockerimage "$@"
     echo "sudo needed to repair file ownership after podman ran: sudo chown -R $USER:$USER ."
     sudo chown -R $USER:$USER .
 
-elif type docker > /dev/null
+elif type docker 2> /dev/null
```
Comment (JonasVautherin) [inline tools/run-docker.sh:11]:
```suggestion
elif type docker > /dev/null 2> /dev/null
```

**Review by JonasVautherin — COMMENTED** [review]
What about a tradeoff? Works for me on `sh`, `zsh` and `bash`.

**Comment by julianoes** [comment]
Hm, you're right. :)

**Pull request merged**